src/main/java/com/iota/iri/network/Node.java

```diff
@@ -515,6 +516,11 @@ private Runnable spawnBroadcasterThread() {
         };
     }
 
+    public void setPauseBetweenTransactions(long nanoseconds) {
```

Comment:
Since this setter doesn't simply assign the parameter to a field but employs logic maybe adding a unit test is a good idea.

Reply:
If you add a unit test @paulhandy I just might approve this change